docs/guide/deployment/resource-directory.md

```diff
@@ -1,4 +1,5 @@
-# Resource directory
+# Resource Directory
+Resource directory exposes resource information published by devices for OCF client to discovery/retrieve from resource shadow.
```

Comment:
Resource Directory keeps and updates in-memory cache of devices which were requested by clients through the plgd gateways.